packages/common/src/types/passwordValidationResult.ts

```diff
@@ -0,0 +1,6 @@
+export type PasswordValidationResult = {
```

Comment:
If you create a new type file, you also need to export it (like in here https://github.com/lightdash/lightdash/blob/main/packages/common/src/index.ts#L133) so we can use it without pointing to the `src`

Reply:
tbh those types are more for models used in DB and APIs, I think you can move this to /index.ts